ql/ql/src/codeql_ql/ast/internal/Predicate.qll

```diff
@@ -209,7 +209,9 @@ module PredConsistency {
         not exists(p0.getAlias())
       ) and
     c > 1 and
-    resolvePredicateExpr(pe, p)
+    resolvePredicateExpr(pe, p) and
+    // parameterized modules are expected to resolve to multiple.
+    not exists(Predicate sig | not exists(sig.getBody()) and resolvePredicateExpr(pe, sig))
```

Comment:
`not exists(sig.getBody)` is the way you're checking that the predicate is part of a parameterized module interface? If so, I think that would well deserve a named predicate or a named class. A name would make this line easier to read, and it would also make it less likely that others in the future have to re-invent/re-discover this logic.